text/0148-sampling-probability.md

```diff
@@ -0,0 +1,209 @@
+# Probability sampling of telemetry events
+
+Specify a foundation for sampling techniques in OpenTelemetry.
+
+## Motivation
+
+In both tracing and metrics, there are widely known techniques for
+sampling events that, when performed correctly, enable ways to lower
+collection costs.  While sampling techniques vary, it is possible
+specify high-level interoperability requirements that producers and
+consumers of sampled data may follow to enable a wide range of
+sampling designs.
+
+## Explanation
+
+Consider a hypothetical telemetry signal in which an API event
+produces a unit of data that has one or more associated numbers.
+Using the OpenTelemetry Metrics data model terminolgy, we have two
+scenarios in which sampling is common.
+
+1. Counter events: Each event represents a count, signifying the change in a sum
+2. Histogram events: Each event represents an individual variable, signifying new membership in a distribution
+
+A Tracing Span event qualifies as both of these cases simultaneously.  It is
+a Counter event (of 1 span) and a Histogram event (of 1 latency measurement).
+
+In Metrics, [Statsd Counter and Histogram events meet this definition](https://github.com/statsd/statsd/blob/master/docs/metric_types.md#sampling).
+
+In both cases, the goal in sampling is to estimate something about the
+population of all events, using only the events that were chosen in
+the sample.  Sampling theory defines various kinds of "estimator",
+algorithms for calculating statistics about the population using just
+the sample data.  For the broad class of telemetry sampling
+application considered here, we need an estimator for the population
+total represented by each individual event.
+
+### Sample count adjustment
+
+The estimated population total divided by the individual event value
+equals the event's _adjusted count_.  The adjusted count is named
+`sample_count` where it appears as a field in a telemetry data type.
```

Comment:
Why not `sample_weight`? Count (a) sounds like an integer, and (b) sounds like we literally received N identical events and compressed into one, while in practice we didn't even look at N-1 events and just assumed that this event represents N because of how it was selected.

Reply:
Considering all the feedback on this OTEP, I removed the recommendation of a specifically named field ("sample_count") in favor of a specific term used to describe the concept, "adjusted count". I reason that this is a count, not a weight, because it tells you how many times to count the event.